lib/cancel/CancelToken.ts

```diff
@@ -0,0 +1,23 @@
+import { Canceler } from "../mock-axios-types";
+
+export default class CancelToken {
+  public static source() {
+    let cancel: Canceler;
+    const token = new CancelToken((c) => (cancel = c));
+
+    return {
+      cancel,
+      token,
+    };
+  }
+
+  constructor(executor: (cancel: Canceler) => void) {
+    executor(function cancel(message) {
+      return;
+    });
+  }
+
+  public throwIfRequested() {
+    return;
```

Comment:
These line is uncovered in the test suite (but should be easy to test)

Reply:
Will add a test that it does not throw.

Reply:
https://github.com/knee-cola/jest-mock-axios/pull/32/commits/1c54152d04f551b46b0795f0ded1457a3fc6415b